tableaudocumentapi/xml_open.py

```diff
@@ -0,0 +1,28 @@
+try:
+    from distutils2.version import NormalizedVersion as Version
+except ImportError:
+    from distutils.version import LooseVersion as Version
+
+import xml.etree.ElementTree as ET
+import zipfile
+
+from tableaudocumentapi import xfile
+
+MIN_SUPPORTED_VERSION = Version("9.0")
+
+
+class VersionNotSupportedException(Exception):
+    pass
+
+
+def xml_open(filename):
```

Comment:
I think this should just go in the xfile module as the functionality is all around opening around zip packaging.

(And that's a great cleanup I didn't get to, thank you!)